src/modules/admin/components/Profile/Profile.tsx

```diff
@@ -31,19 +26,13 @@ const Profile = ({ colony }: Props) => (
         <Tab>
```

Comment:
This shouldn't have any tabs, it should now **just** be the profile component. If you look at the figma spec, you'll notice this is simply a heading now:

<img width="1516" alt="Screen Shot 2019-09-25 at 12 57 16 PM" src="https://user-images.githubusercontent.com/3052635/65627136-583c2000-df94-11e9-8b38-765767bce775.png">